### PR TITLE
revise preqs for install docs

### PR DESCRIPTION
**Pull request by geoffcline**
**Is this a bug fix or adding new feature?**
users were confused about the preq steps about checking feature gates.

**What is this PR about? / Why do we need it?**
these need for enabling these gates is largely outdated. it is also clearer to discribe the version directly. 

**What testing is done?**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
 # Installation
 
 ## Prerequisites
+
+* Kubernetes Version >= 1.20
+  * Versions 1.18 - 1.19 may require the `VolumeSnapshotDataSource` feature gate be enabled on `kubelet`.
```
Comment (ConnorJC3) [inline docs/install.md:6]: 1.18 and 1.19 are long EOL and not even supported by EKS anymore, maybe we can finally just drop the extra note?

File: docs/install.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
 # Installation
 
 ## Prerequisites
+
+* Kubernetes Version >= 1.20
+  * Versions 1.18 - 1.19 may require the `VolumeSnapshotDataSource` feature gate be enabled on `kubelet`. 
+
+* If you are using a self managed cluster, ensure the flag `--allow-privileged=true` for `kube-apiserver`.
+
+* Important: If you intend to use the csi-snapshotter functionality, the [CSI Snapshotter](https://github.com/kubernetes-csi/external-snapshotter). **must be installed before* the EBS CSI driver.
```
Comment (ConnorJC3) [inline docs/install.md:10]: `before*` -> `before**`

File: docs/install.md (hunk 1 above)
Comment (geoffcline) [reply docs/install.md:6]: I agree in spirit, but you can still create new 1.19 clusters on EKS. 

https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

Comment (ConnorJC3) [reply docs/install.md:6]: Screenshot_20220916-160653

When I go to create a cluster in the console, it doesn't give 1.19 as an option. Is it possible those docs are out of date? Or maybe it's possible only via the CLI/API for some reason?

Comment (geoffcline) [reply docs/install.md:6]: the docs are out of date...... so yes now i'm fine removing it.

**Review by ConnorJC3 — APPROVED** [review]
lgtm 👍

**Review by torredil — COMMENTED** [review]
/lgtm 
Thanks, will approve this PR once the commits have been squashed.

**Comment by geoffcline** [comment]
It has been squashed @torredil :) thanks

**Review by torredil — APPROVED** [review]
/lgtm

**Pull request merged**